04-simulate-an-election/election.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+from random import random
+A = "a"
+B = "b"
+
+def elect_in_region(election, chances):
```

Comment:
Election is a bad name. use `result` instead which goes along nicely with the `random_result` you pass it in `simulate_a_single_election`.